core/modules/obsolete/TableMerger.cc

```diff
@@ -102,9 +104,9 @@ void inplaceReplace(std::string& s, std::string const& old,
     if(rplc.size() < rplcSize) { // do padding for in-place
         rplc += std::string(rplcSize - rplc.size(), ' ');
     }
-    //LOGF_DEBUG("rplc %1% old=%2%" % rplc % s.substr(pos-1, rplcSize+2));
+    //LOGS(_log, LOG_LVL_DEBUG("rplc " << rplc << " old=" << s.substr(pos-1, rplcSize+2));
```

Comment:
I didn't delete these lines because I think it saves time when someone want to debug these specific parts. Indeed I didn't test the commented out lines. Let's discuss tomorrow.